awx/main/models/ha.py

```diff
@@ -114,6 +114,11 @@ class Instance(HasPolicyEditsMixin, BaseModel):
         editable=False,
         help_text=_('Last time instance ran its heartbeat task for main cluster nodes. Last known connection to receptor mesh for execution nodes.'),
     )
+    health_check_started = models.DateTimeField(
+        null=True,
+        editable=False,
+        help_text=_("The last time a health check was initiated on this instance."),
```

Comment:
...only for execution nodes

Reply:
Why?  We can trigger the `cluster_node_health_check` from the endpoint as well.